.github/workflows/master.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         java-version: '11'
     - name: install R
       run: |
-        echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' | sudo tee -a /etc/apt/sources.list
+        echo 'deb https://cloud.r-project.org/bin/linux/ubuntu/bionic-cran35/' | sudo tee -a /etc/apt/sources.list
```

Comment:
The old one is still correct.
- https://wiki.debian.org/SourcesList

Reply:
Yes, let me run some builds and try here.